Assets/Scripts/Game/Entities/PlayerEntity.cs

```diff
@@ -450,7 +450,7 @@ public override void Update(DaggerfallEntityBehaviour sender)
             HandleStartingCrimeGuildQuests();
 
             // Reset surrender to guards dialogue if no guards are nearby
-            if (haveShownSurrenderToGuardsDialogue && GameManager.Instance.HowManyEnemiesOfType(MobileTypes.Knight_CityWatch, true) == 0)
+            if (haveShownSurrenderToGuardsDialogue && GameManager.Instance.HowManyEnemiesOfType(MobileTypes.Giant, true) == 0)
```

Comment:
Did you mean to leave these changes in the PR? Looks like testing stuff to me... but i could be wrong

Reply:
Haha, yeah. I was testing some giants to make sure they could fit under the mages guild sign because I had noticed that was a case where the player can get through but AI characters couldn't. Let me fix that.